### PR TITLE
Add a new config `server.bolt.additional_listen_addresses`

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1726.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1726.surge.sh/operations-manual/5/configuration/configuration-settings)

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 1]
```diff
@@ -1249,6 +1249,20 @@ a|A socket address in the format of `hostname:port`, `hostname`, or `:port`. If
...
+[[config_server.bolt.additional_listen_addresses]]
+=== `server.bolt.additional_listen_addresses`
+
+.server.bolt.additional_listen_addresses
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|Additional addresses the connector should bind to.
+|Valid values
+a|A socket address in the format of `hostname:port`, `hostname`, or `:port`.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:1262]:
```suggestion
a|A comma-separated set where each element is a socket address in the format of hostname:port, hostname, or :port.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1726.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1726.surge.sh/operations-manual/5/configuration/configuration-settings)

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 2]
```diff
@@ -1249,6 +1249,20 @@ a|A socket address in the format of `hostname:port`, `hostname`, or `:port`. If
...
+
+.server.bolt.additional_listen_addresses
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|Additional addresses the connector should bind to.
+|Valid values
+a|A comma-separated set where each element is a socket address in the format of hostname:port, hostname, or :port.
+|Default value
+m|+++127.0.0.1:0+++
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:1264]:
```suggestion
m|
```

File: modules/ROOT/pages/configuration/configuration-settings.adoc [hunk 3]
```diff
@@ -1249,6 +1249,20 @@ a|A socket address in the format of `hostname:port`, `hostname`, or `:port`. If
...
+[[config_server.bolt.additional_listen_addresses]]
+=== `server.bolt.additional_listen_addresses`
+
+.server.bolt.additional_listen_addresses
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|Additional addresses the connector should bind to.
+|Valid values
+a|A comma-separated set where each element is a socket address in the format of hostname:port, hostname, or :port.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/configuration/configuration-settings.adoc:1262]:
```suggestion
a|A comma-separated set where each element is a socket address in the format of `hostname:port`, `hostname`, or `:port`.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1726.surge.sh

Updated pages:
- [Configuration settings](https://neo4j-docs-operations-1726.surge.sh/operations-manual/5/configuration/configuration-settings)

**Pull request merged**